redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -142,7 +143,7 @@ def _get_conn(self, instance):
                 self.connections[key] = redis.Redis(**connection_params)
 
             except TypeError:
-                msg = "You need a redis library that supports authenticated connections. Try sudo easy_install redis."
+                msg = "You need a redis library that supports authenticated connections. Try `pip install redis`."
```

Comment:
why remove the sudo?

Reply:
It's unsafe to mess with system packages